stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -419,6 +420,38 @@ public Token createTokenSynchronous(final Card card, String publishableKey)
     }
 
     /**
+     *  Starts polling the {@link Source} object with the given ID. For payment methods that require
+     *  additional customer action (e.g. authorizing a payment with their bank), polling
+     *  allows you to determine if the action was successful. Polling will stop and the
+     *  provided callback will be called once the source's status is no longer
+     *  {@link Source#PENDING}, or if the given timeout is reached and the source is still
+     *  `pending`. If polling stops due to an error, the callback will be fired with the latest
+     *  retrieved source and the error.
+     *
+     *  Note that if a poll is already running for a source, subsequent calls to `pollSource`
```

Comment:
Also, ci-bot is failing pretty hard -- you clearly put the approved tag on this, but it's not showing as approved.